website/.github/ISSUE_TEMPLATE/communication-suggestion.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Can see this potentially overlapping with feature request or bug report so remove this.